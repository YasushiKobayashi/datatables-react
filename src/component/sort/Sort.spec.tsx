import * as React from 'react';
import * as Enzyme from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';

import Sort from './Sort';

import Order from '../../models/enum/Order';

Enzyme.configure({ adapter: new Adapter() });

const base = {
  order: Order.asc,
};

describe('Sort', () => {
  it('renders correctly', () => {
    const props = Object.assign({}, base);
    const tree = Enzyme.shallow(<Sort {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
