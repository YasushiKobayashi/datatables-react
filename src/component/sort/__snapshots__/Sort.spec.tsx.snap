// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sort renders correctly 1`] = `ShallowWrapper {}`;
